### ISSUE TITLE
addToSpace slash command not working

### ISSUE THREAD

**Issue by chagai95** [vector-im/element-android#6844]
### Steps to reproduce

1. Go to a room.
2. Run /addToSpace command:

![image](https://user-images.githubusercontent.com/31655082/184904799-2c8ee3c4-9b43-4639-93d6-f827341945e6.png)

#### What did you expect?
The room to be added to that space.

#### What happened instead?
The error appeared

### Your phone model

Xiaomi 11

### Operating system version

Android 12

### Application version and app store

1.4.32

### Homeserver

matrix.org

### Will you send logs?

No

### Are you willing to provide a PR?

Yes

**Comment by ouchadam** [comment]
am also able to confirm with a unit test in `CommandParserTest`

```kotlin
@Test
fun parseAddToSpaceCommand() {
    test("/addToSpace !my-space-id", ParsedCommand.AddToSpace("!my-space-id"))
}
```

looks like the `CommandParser` is only expecting 1 segment to the command when it should be 2

**Comment by ouchadam** [comment]
fixed in 1.4.36
